jormungandr/src/log/stream.rs

```diff
@@ -76,24 +82,25 @@ impl<'a, S: Debug, M: Debug> Debug for LoggingStream<'a, S, M> {
 
 impl<'a, S, M> Stream for LoggingStream<'a, S, M>
 where
-    S: Stream,
-    S::Item: Debug,
-    M: Display,
+    S: Stream + Unpin,
+    S::Item: Debug + Unpin,
+    M: Display + Unpin,
```

Comment:
I hope this is acceptable sacrifice for being able to use `Pin::into_inner` for convenience. Not every stream is `Unpin`. On the other hand, we hardly use this adapter and we should transition to `tracing` as our logging framework in the future, so it's OK to just remove the whole thing.

Look into [pin-project](https://crates.io/crates/pin-project) as a nice way to do `Pin` projections.

Reply:
Thanks, will try it!

Reply:
Worked well for logs. Not sure if will work at all for `FireForgetSchedulerFuture`, but will look how to simplify that one too.